spacy/language.py

```diff
@@ -1825,7 +1825,7 @@ def from_config(
         # Later we replace the component config with the raw config again.
         interpolated = filled.interpolate() if not filled.is_interpolated else filled
         pipeline = interpolated.get("components", {})
-        sourced = util.get_sourced_components(interpolated)
+        sourced = util.get_sourced_components(interpolated)  # noqa: F841
```

Comment:
This looks like a leftover from https://github.com/explosion/spaCy/pull/12701. For now, let's remove the noqa from the code and just fix the typo's in this PR.